config.tpl.yml

```diff
@@ -1,4 +1,8 @@
 ---
+# yaml-language-server: $schema=config.schema.json
+
+# name of the cluster
+project_name: az-hop
```

Comment:
Note: this is something we found useful when administering multiple clusters. I can move it out of the PR if you prefer

Reply:
Yes we have added a cluster_name for slurm to prepare federation. Please work on a PR for a better separation of updates.